mantis-publish/mantis-publish-core/src/test/java/io/mantisrx/publish/internal/discovery/MantisJobDiscoveryTest.java

```diff
@@ -121,8 +121,8 @@ public void testJobDiscoveryFetch() throws IOException {
         assertEquals(jobSchedulingInfo.getWorkerAssignments().get(1).getHosts().get(1).getHost(), currentJobWorkers.get().getIngestStageWorkers().getWorkers().get(0).getHost());
     }
 
+    @Disabled("another flaky test")
```

Comment:
Do we know what makes this test flaky? Is it inherent to the code or the test itself?

Reply:
Actually I'm withdrawing this comment from this review. This PR didn't introduce this test, and is just renaming the annotation from `@Ignore` to `@Disabled`.